Makefile

```diff
@@ -99,4 +102,4 @@ build/docker/push:
 build/docker/untag:
 	echo "docker rmi brigadehub/$(PACKAGENAME):release"
 
-.PHONY: start lint test db install build link
+.PHONY: start lint test db install build link .env
```

Comment:
Actually, `.env` is **not** "phony", it's a real file, so I don't think you want this here.